plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/XmlSourceValidator.java

```diff
@@ -111,19 +109,11 @@ void validate(IReporter reporter, IFile source, byte[] bytes) throws IOException
   @Override
   public void setInitializationData(IConfigurationElement config, String propertyName, Object data)
       throws CoreException {
-    try {
-      if (data == null || !(data instanceof String)) {
-        throw new CoreException(StatusUtil.error(getClass(), "Data must be a class name"));
-      }
-      String className = (String) data;
-      Class<?> clazz = Class.forName(className);
-      Constructor<?> constructor = clazz.getConstructor();
-      this.setHelper((XmlValidationHelper) constructor.newInstance(new Object[] {}));
-    } catch (ClassNotFoundException | NoSuchMethodException | SecurityException
-        | InstantiationException | IllegalAccessException | IllegalArgumentException
-        | InvocationTargetException ex) {
-      logger.log(Level.SEVERE, ex.getMessage());
+    if (data == null || !(data instanceof String)) {
```

Comment:
I'm tempted to remove `data == null`, as `!(data instanceof String)` subsumes it.

Reply:
tests are failing so I'll add this.